source/dub/dub.d

```diff
@@ -65,6 +65,7 @@ static this()
 
 /// The URL to the official package registry.
 enum defaultRegistryURL = "http://code.dlang.org/";
+enum fallbackRegistryURL = "https://code-mirror.dlang.io/";
```

Comment:
> Heroku Deploy is a bit more difficult to setup due to rejectedsoftware/vibe.d#1785, but I will get around this as well.

-> https://dub-registry.herokuapp.com/

With https://github.com/dlang/dub-registry/pull/231 setting up a local mirror should be absolutely easy (and doesn't even require a CLI nor money).